src/controllers/issuer.ts

```diff
@@ -287,36 +287,37 @@ export class IssuerController {
       if (request.body.didDocument) {
         updatedDocument = request.body.didDocument
       } else if (did && (service || verificationMethod || authentication)) {
-        const resolvedResult = await new Identity(response.locals.customerId).agent.resolveDid(did)
-        if(!resolvedResult?.didDocument || resolvedResult.didDocumentMetadata.deactivated) {
+        const body = await (await fetch(`${process.env.RESOLVER_URL}/${did}`)).json()
+        const resolvedResult: DIDResolutionResult = JSON.parse(body)
+        if (!resolvedResult?.didDocument || resolvedResult.didDocumentMetadata.deactivated) {
           return response.status(StatusCodes.BAD_REQUEST).send({
```

Comment:
I don't think that here the error needs to be BAD_REQUEST. Mostly BAD_GATEWAY cause such request is not handled by CredentialService side but by did-resolver.

Reply:
Done! ✅